chihuahua/assetlist.json

```diff
@@ -104,6 +104,31 @@
       "images": [
         {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/woof.png"
+    },
```

Comment:
on both assetlists, you also need a closing bracket for the previous asset's image, and then a closing bracket for the previous asset's images array, which go before the previous asset's closing bracket